doc/operator-scope.md

```diff
@@ -1,9 +1,30 @@
-# Operator SDK: Operator Scope
+## Operators and CRD scope with Operator SDK
+
+- [Operator scope](#operator-scope)
+  - [When a operator namespaced-scoped should be used?](#when-a-operator-namespaced-scoped-should-be-used)
+  - [When a operator cluster-scoped should be used?](#when-a-operator-cluster-scoped-should-be-used)
+  - [Changes required in the scaffold for cluster scoped operator](#changes-required-in-the-scaffold-for-cluster-scoped-operator)
+  - [Example for cluster scoped operator](#example-for-cluster-scoped-operator)
+- [CRD scope](#crd-scope)
+  - [When CRD cluster-scoped should be used?](#when-crd-cluster-scoped-should-be-used)
+  - [Example for change the CRD scope from namespace to cluster](#example-for-change-the-crd-scope-from-namespace-to-cluster)
 
 A namespace-scoped operator watches and manages resources in a single namespace, whereas a cluster-scoped operator watches and manages resources cluster-wide. Namespace-scoped operators are preferred because of their flexibility. They enable decoupled upgrades, namespace isolation for failures and monitoring, and differing API definitions.
 
 However, there are use cases where a cluster-scoped operator may make sense. For example, the [cert-manager](https://github.com/jetstack/cert-manager) operator is often deployed with cluster-scoped permissions and watches so that it can manage issuing certificates for an entire cluster.
 
+#### When an operator namespaced-scoped should be used?
```

Comment:
If you remove the ?, this (and line 22) will be fine. If you prefer to leave the ?, it should be:

"When should a namespace-scoped operator be used?"